lib/src/widgets/custom_image_crop_widget.dart

```diff
@@ -40,9 +40,27 @@ class CustomImageCrop extends StatefulWidget {
   /// custom one
   final CustomPaint Function(Path) drawPath;
 
+  /// Whether to allow the image to be rotated.
+  final bool canRotate;
+
+  /// Determines whether scaling gesture is disabled.
+  ///
+  /// By default, scaling is enabled. Set [disableScale] to `true`
+  /// to disable scaling, which can be useful for loading
+  final bool disableGestureScale;
```

Comment:
Use consistent field names. 
`canRotate` is different form` disableGestureScale`

suggestion:
`canScale`